tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/wizard/NewLiferayModuleProjectWizard.java

```diff
@@ -0,0 +1,333 @@
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ *******************************************************************************/
+package com.liferay.ide.project.ui.wizard;
+
+import com.liferay.ide.core.util.CoreUtil;
+import com.liferay.ide.project.core.ProjectCore;
+import com.liferay.ide.project.core.model.NewLiferayModuleProjectOp;
+import com.liferay.ide.project.core.model.ProjectName;
+import com.liferay.ide.project.ui.IvyUtil;
+import com.liferay.ide.project.ui.ProjectUI;
+import com.liferay.ide.sdk.core.ISDKConstants;
+import com.liferay.ide.ui.LiferayPerspectiveFactory;
+import com.liferay.ide.ui.util.UIUtil;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.ant.internal.ui.model.AntProjectNode;
+import org.eclipse.ant.internal.ui.model.AntProjectNodeProxy;
+import org.eclipse.ant.internal.ui.views.AntView;
+import org.eclipse.core.resources.IFile;
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.WorkspaceJob;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IConfigurationElement;
+import org.eclipse.core.runtime.IExtension;
+import org.eclipse.core.runtime.IExtensionRegistry;
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.Platform;
+import org.eclipse.core.runtime.Status;
+import org.eclipse.jdt.internal.ui.util.CoreUtility;
+import org.eclipse.jface.dialogs.IMessageProvider;
+import org.eclipse.jface.viewers.IStructuredSelection;
+import org.eclipse.jface.viewers.StructuredSelection;
+import org.eclipse.jface.wizard.IWizardPage;
+import org.eclipse.jface.wizard.WizardDialog;
+import org.eclipse.sapphire.ElementList;
+import org.eclipse.sapphire.ui.def.DefinitionLoader;
+import org.eclipse.sapphire.ui.forms.FormComponentPart;
+import org.eclipse.sapphire.ui.forms.swt.SapphireWizard;
+import org.eclipse.sapphire.ui.forms.swt.SapphireWizardPage;
+import org.eclipse.swt.widgets.Display;
+import org.eclipse.swt.widgets.Shell;
+import org.eclipse.ui.INewWizard;
+import org.eclipse.ui.IPageLayout;
+import org.eclipse.ui.IViewPart;
+import org.eclipse.ui.IWorkbench;
+import org.eclipse.ui.IWorkbenchWizard;
+import org.eclipse.ui.IWorkingSet;
+import org.eclipse.ui.PlatformUI;
+import org.eclipse.ui.navigator.CommonViewer;
+import org.eclipse.ui.wizards.newresource.BasicNewProjectResourceWizard;
+import org.eclipse.ui.wizards.newresource.BasicNewResourceWizard;
+import org.eclipse.wst.web.internal.DelegateConfigurationElement;
+
+
+/**
+ * @author Simon Jiang
+ */
+@SuppressWarnings( "restriction" )
+public class NewLiferayModuleProjectWizard extends SapphireWizard<NewLiferayModuleProjectOp>
```

Comment:
All module wizard stuff should go in a new package. "com.liferay.ide.project.ui.modules"